Sources/SwiftDocC/Utility/Synchronization.swift

```diff
@@ -97,16 +97,16 @@ public class Synchronized<Value> {
 /// lock.sync { myVar = 1 }
 /// let result = lock.sync { return index["key"] }
 /// ```
-typealias Lock = Synchronized<Void>
+package typealias Lock = Synchronized<Void>
 
-public extension Lock {
```

Comment:
Is it safe to mark this extension as non-public? Could some client package be using it?

Reply:
Yes, it's safe because it's extending an internal type so its API was never actually available to the public.

Because `Lock` was defined as `typealias Lock = Synchronized<Void>` (no access modifier) it was never publicly available so a public extension to Lock could never exceed the Lock types internal access. I feel that there should have been a warning about this but there wasn't.

Reply:
I only found this because I tried to use this API from SwiftDocCUtilities and got a build error that it could't be accessed because it was internal.